app.css

```diff
@@ -1,7 +1,119 @@
+body {
+  font-family: sans-serif;
+  font-size: 16px;
+}
+
 .timeline {
+  position: relative;
   list-style: none;
   margin-left: 200px;
-  border-left: #575bb4 solid 4px;
+  border: 0;
+  border-left: #92cbe0 solid 4px;
   border-image:
-      linear-gradient(to bottom, #575bb4, #8e92f5) 1 100%;
+      linear-gradient(to bottom, #92cbe0, #287f9f) 1 100%;
+}
+/* Timeline Article Styling */
+.article {
+  background: #92cbe0;
+  color: white;
+  width: 500px;
+  padding: 20px;
+  border-radius: .5em;
+  margin-bottom: 15px;
+}
+
+.article:nth-of-type(2n) {
+  background: #287f9f;
+}
+
+/* Article Font Styling */
+h1 {
+  font-size: 1.3rem;
+  border-bottom: 1px rgb(234, 238, 245) solid;
+  padding-bottom: 5px;
+}
+.body {
+  font-size: 14px;
+  font-weight: lighter;
+}
+
+/* Timeline Indicator Items */
+
+.section  {
+  position: absolute;
+  float: left;
+  right: 105%;
+}
+
+.section .date {
+  font-size: 10px;
+  line-height: 10px;
+  color: #c8c3c3;
+  margin-bottom: 0;
+  margin-top: -22px;
+  padding-left: 40px;
+}
+
+.section .time {
+  font-size: 30px;
+  line-height: 30px;
+  color: #92cbe0;
+  margin-top: 0;
+}
+
+.section .time:nth-of-type(2n) {
+  color: #287f9f;
+}
+
+ .section i {
+  position: absolute;
+  left: 105%;
+  float: left;
+  display: inline-block;
+  vertical-align: center;
+  height: 34px;
+  width: 34px;
+  border: 6px #92cbe0 solid;
+  border-radius: 100%;
+  text-align: center;
+  line-height: 34px;
+  background: #287f9f;
+  color: white;
+}
+
+.fa-mobile {
+  font-size: 27.4px;
+  bottom: 70%;
+}
+
+.fa-desktop {
+  font-size: 18px;
+  bottom: 63%;
+}
+
+.fa-star-o {
+  font-size: 26px;
+  bottom: 70%;
+}
+
+/* Triangle Styling */
+.triangle {
+  overflow: visible;
+  position: relative;
+}
+
+.triangle::after {
+  content: "";
+  position: absolute;
+  right: 103%;
+  bottom: 106%;
+  border-style: solid;
+  border-width: 7.5px 13.0px 7.5px 0;
+  border-color: transparent #92cbe0 transparent transparent;
+}
+
+.triangle::after:nth-of-type(2n) {
```

Comment:
See the comment above: the same thing happens for these triangles: https://github.com/pixiestix826/04-Timeline/pull/1/files#r43694414